docs/getting-started/advanced-installation.md

```diff
@@ -17,13 +17,30 @@ If you have problem with downloading, use this:
 Invoke-WebRequest -UseBasicParsing "https://raw.githubusercontent.com/spicetify/spicetify-cli/master/install_curl.ps1" | Invoke-Expression
 ```
 
+### Chocolatey
+
+Follow this guide: https://chocolatey.org/packages/spicetify-cli
+
 ### Scoop
 
-Follow this guide: https://github.com/zwxi/Scoop-Spotify
+```powershell
+scoop install spicetify-cli
+```
 
-### Chocolatey
+### Note for Windows users
```

Comment:
I'd remove this as we are already in the "windows" section.

Reply:
I was being consistent with the `Note for Linux users` section below but seems like that is under the `Linux and MacOS` section. I'll rename this to `Notes`.

Reply:
I was being consistent with the `Note for Linux users` section below but seems like that is under the `Linux and MacOS` section. I'll rename this to `Notes`.

Reply:
I like it, thanks!